### ISSUE TITLE
Strip BOM in boundary file

### ISSUE THREAD

**Issue by michaelsteffen** [mapbox/mapping#338]
worldcities.json appears to include a BOM. Can we remove and commit a clean version here?

@ramyaragupathy @planemad

**Comment by ramyaragupathy** [comment]
Picking this for clean-up now @planemad

**Comment by ramyaragupathy** [comment]
@planemad - Done with my round of clean up. Please go through the data file for additional modifications.

#### Changes Done:
- Remove duplicate entries 
- Removed Oakland & Alameda, since they're not intended for the priority list
- Reordered the content by continent (EU, Asia & NA). Within each continent cities in a particular country/region are grouped together. 
- Only NA & EU cities have distinct exterior/core boundaries, arranged exterior & core boundaries of every city to stay one after another

<details>
<summary><h3>Order of cities within the data file</h3></summary>

properties/label | properties/is_in | properties/country | properties/state
-- | -- | -- | --
Lyon_exterior | europe | france | rhone-alpes
Lyon_core | europe | france | rhone-alpes
Nice_exterior | europe | france | provence-alpes-cote-d-azur
Nice_core | europe | france | provence-alpes-cote-d-azur
Marseille_exterior | europe | france | provence-alpes-cote-d-azur
Marseille_core | europe | france | provence-alpes-cote-d-azur
Paris_exterior | europe | france | ile-de-france
Paris_core | europe | france | ile-de-france
Toulouse_exterior | europe | france | midi-pyrenees
Toulouse_core | europe | france | midi-pyrenees
Berlin_GE | europe | germany | berlin
Frankfurt_GE | europe | germany | hessen
Munich_GE | europe | germany | bayern
Wolfsburg_GE | europe | germany | niedersachsen
Stuttgart_GE | europe | germany | baden-wuerttemberg
Singapore_exterior | asia | malaysia-singapore-brunei |  
JohorBahru_exterior | asia | malaysia-singapore-brunei |  
KotaKinabalu_exterior | asia | malaysia-singapore-brunei |  
KualaLumpur_exterior | asia | malaysia-singapore-brunei |  
Putrajaya_exterior | asia | malaysia-singapore-brunei |  
Kuching_exterior | asia | malaysia-singapore-brunei |  
Cyberjaya_exterior | asia | malaysia-singapore-brunei |  
Malacca_exterior | asia | malaysia-singapore-brunei |  
Penang_exterior | asia | malaysia-singapore-brunei |  
HoChiMinhCity_exterior | asia | vietnam |  
Halong_exterior | asia | vietnam |  
Hanoi_exterior | asia | vietnam |  
Bandung_exterior | asia | indonesia |  
Pekanbaru_exterior | asia | indonesia |  
Surabaya_exterior | asia | indonesia |  
Padang_exterior | asia | indonesia |  
Jakarta_exterior | asia | indonesia |  
Pattaya_exterior | asia | thailand |  
Phuket_exterior | asia | thailand |  
Bangkok_exterior | asia | thailand |  
ChiangRai_exterior | asia | thailand |  
Chiang_exterior | asia | thailand |  
IloiloCity_exterior | asia | philippines |  
Manila_exterior | asia | philippines |  
CebuCity_exterior | asia | philippines |  
CagayanDeOro_exterior | asia | philippines |  
DavaoCity_exterior | asia | philippines |  
Bacolod_exterior | asia | philippines |  
Baguio_exterior | asia | philippines |  
SiemReap_exterior | asia | cambodia |  
PhnomPenh_exterior | asia | cambodia |  
Mandalay_exterior | asia | myanmar |  
Yangon_exterior | asia | myanmar |  
Calgary_CA | north-america | canada | alberta
Toronto_CA | north-america | canada | ontario
Ottawa_CA | north-america | canada | ontario
Vancouver_CA | north-america | canada | british-columbia
Montreal_CA | north-america | canada | quebec
Los_Angeles_exterior_CA | north-america | us | california
SFbay_exterior_CA | north-america | us | california
SFCity_exterior_CA | north-america | us | california
Phoenix_exterior_AZ | north-america | us | arizona
ElPaso_exterior_TX | north-america | us | texas
Buffalo_exterior_NY | north-america | us | new-york
Buffalo_core_NY | north-america | us | new-york
Columbus_exterior_OH | north-america | us | ohio
Columbus_core_OH | north-america | us | ohio
Salt_Lake_exterior_UT | north-america | us | utah
Salt_Lake_core_UT | north-america | us | utah
NewYork_exterior_NY | north-america | us | new-york
NewYork_core_NY | north-america | us | new-york
Houston_exterior_TX | north-america | us | texas
Houston_core_TX | north-america | us | texas
Philadelphia_exterior_PA | north-america | us | pennsylvania
Philadelphia_core_PA | north-america | us | pennsylvania
Seattle_exterior_WA | north-america | us | washington
Seattle_core_WA | north-america | us | washington
Atlantic_exterior_NJ | north-america | us | new-jersey
Atlantic_core_NJ | north-america | us | new-jersey
Charlotte_exterior_NC | north-america | us | north-carolina
Charlotte_core_NC | north-america | us | north-carolina
Portland_exterior_OR | north-america | us | oregon
Portland_core_OR | north-america | us | oregon
Milwaukee_exterior_WI | north-america | us | wisconsin
Milwaukee_core_WI | north-america | us | wisconsin
Denver_exterior_CO | north-america | us | colorado
Denver_core_CO | north-america | us | colorado
Jacksonville_exterior_FL | north-america | us | florida
Jacksonville_core_FL | north-america | us | florida
Austin_exterior_TX | north-america | us | texas
Austin_core_TX | north-america | us | texas
Oklahoma_exterior_OK | north-america | us | oklahoma
Oklahoma_core_OK | north-america | us | oklahoma
Minneapolis-StPaul_exterior_MN | north-america | us | minnesota
Minneapolis-StPaul_core_MN | north-america | us | minnesota
Louisville_exterior_KY | north-america | us | kentucky
Louisville_core_KY | north-america | us | kentucky
Chicago_exterior_IL | north-america | us | illinois
Chicago_core_IL | north-america | us | illinois
Nashville_exterior_TN | north-america | us | tennessee
Nashville_core_TN | north-america | us | tennessee
Detroit_exterior_MI | north-america | us | michigan
Detroit_core_MI | north-america | us | michigan
Memphis_exterior_TN | north-america | us | tennessee
Memphis_core_TN | north-america | us | tennessee
Washington_exterior_DC | north-america | us | district-of-columbia
Washington_core_DC | north-america | us | district-of-columbia
Orlando_exterior_FL | north-america | us | florida
Orlando_core_FL | north-america | us | florida
Boston_exterior_MA | north-america | us | massachusetts
Boston_core_MA | north-america | us | massachusetts
Atlanta_exterior_GA | north-america | us | georgia
Atlanta_core_GA | north-america | us | georgia
Kansas_exterior_MO | north-america | us | missouri
Kansas_core_MO | north-america | us | missouri
Knoxville_exterior_TN | north-america | us | tennessee
Knoxville_core_TN | north-america | us | tennessee
Baltimore_exterior_MD | north-america | us | maryland
Baltimore_core_MD | north-america | us | maryland
San_Antonio_exterior_TX | north-america | us | texas
San_Antonio_core_TX | north-america | us | texas
Las_Vegas_exterior_NV | north-america | us | nevada
Las_Vegas_core_NV | north-america | us | nevada
Miami_exterior_FL | north-america | us | florida
Miami_core_FL | north-america | us | florida
Fort_Worth_Dallas_exterior_TX | north-america | us | texas
Fort_Worth_Dallas_core_TX | north-america | us | texas
Indianapolis_exterior_IN | north-america | us | indiana
Indianapolis_core_IN | north-america | us | indiana
San_Diego_exterior_CA | north-america | us | california
San_Diego_core_CA | north-america | us | california
Pittsburgh_exterior_PA | north-america | us | pennsylvania
Pittsburgh_core_PA | north-america | us | pennsylvania


</details>

**Comment by planemad** [comment]
Nice work @ramyaragupathy, this looks clean for now.

**Comment by michaelsteffen** [comment]
👍 thanks @ramyaragupathy

**Comment by michaelsteffen** [comment]
@ramyaragupathy still seeing a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) at the beginning of the file. Can you check?

**Comment by ramyaragupathy** [comment]
Noted one dot char in the first line.

![image](https://user-images.githubusercontent.com/12103383/32012832-de701036-b9d6-11e7-81f0-6794875238e8.png)

Unsure of how this was added, since I formatted the entire content within a text editor. This doesn't appear in the raw format or geojson preview. This shows up only while clicking `edit` link within the Github UI. @michaelsteffen  would be great to learn your technique to spot this!
